presto-docs/src/main/sphinx/release/release-0.205.rst

```diff
@@ -0,0 +1,72 @@
+=============
+Release 0.205
+=============
+
+General Changes
+---------------
+
+* Fix parsing of row type signatures with spaces and types containing spaces.
+* Fix distributed planning failure for complex queries when using bucketed execution.
+* Fix a bug that triggers checks for ``SELECT`` privileges for tables in ``DELETE`` queries.
+  This bug only affects connectors that implement ``checkCanSelectFromColumns()``.
+* Fix a bug in enforcing column access controls when querying views. This bug
+  only affects connectors that implement the column access control methods.
+* Fix a bug in :func:`ST_ExteriorRing` to make it only work with polygons.
```

Comment:
Avoid "fix a bug". How about (not sure I have the terminology correct)
```
* Fix :func:`ST_ExteriorRing` to only accept polygons.
  Previously, it erroneously accepted other geometries.

Reply:
/cc @jwass @mbasmanova

Reply:
@electrum @nezihyigitbasi I like David's version.